lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -176,10 +176,16 @@ sub _link_asset ($asset, $pooldir) {
     # Try to use hardlinks first and only fall back to symlinks when that fails,
     # to ensure that assets cannot be purged early from the pool even if the
     # cache service runs out of space
-    eval { link($asset, $target) or die qq{Cannot create link from "$asset" to "$target": $!} };
-    if (my $err = $@) {
+    # If the given asset is a symlink itself, to not hardlink a symlink
```

Comment:
"to not hardlink a symlink"? I guess s/to/do?

Reply:
argh, yes. fixed